packages/@react-facet/core/src/types.ts

```diff
@@ -13,6 +13,15 @@ export interface EqualityCheck<T> {
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export type Value = string | number | boolean | undefined | null | [] | Record<string, any>
```

Comment:
Is this type to broad? The comment above says that functions is not supported, but Record<string, any> includes functions

<img width="356" alt="CleanShot 2023-03-06 at 14 09 27@2x" src="https://user-images.githubusercontent.com/1597781/223119359-857046de-fdef-4abe-8310-f9c5413b88eb.png">

I created another type `ValueWithoutFunction` that does not accept functions